xml/System.Drawing/Graphics.xml

```diff
@@ -8661,11 +8684,12 @@ The <see cref="T:System.Drawing.Graphics" /> object has a transform applied othe
         <Parameter Name="radius" Type="System.Drawing.SizeF" Index="2" FrameworkAlternate="windowsdesktop-9.0;dotnet-plat-ext-9.0;net-9.0" />
       </Parameters>
       <Docs>
-        <param name="pen">To be added.</param>
+        <param name="pen">The <see cref="T:System.Drawing.Pen" /> to draw the outline with.</param>
         <param name="rect">To be added.</param>
         <param name="radius">To be added.</param>
```

Comment:
@lonitra Can you include descriptions for these two parameters? Otherwise this API will continue showing up as undocumented in the report.